playgrounds/app/src/components/Editor.tsx

```diff
@@ -1,26 +1,23 @@
-import { interpolate, interpolateColors, Easing } from 'remotion'
-import { encode } from 'modern-gif'
-import workerUrl from 'modern-gif/worker?url'
 import 'shiki-magic-move/dist/style.css'
+
 import {
-  ComboboxItem,
-  ComboboxItemLabel,
-  ComboboxItemIndicator,
+  Combobox,
+  ComboboxContent,
   ComboboxControl,
   ComboboxInput,
+  ComboboxItem,
+  ComboboxItemIndicator,
+  ComboboxItemLabel,
   ComboboxTrigger,
-  ComboboxContent,
-  Combobox,
 } from '~/components/ui/combobox'
-import { Button } from '~/components/ui/button'
-import { Tabs, TabsContent, TabsList, TabsTrigger } from '~/components/ui/tabs'
 import {
-  TextField,
-  TextFieldInput,
-  TextFieldLabel,
-  TextFieldTextArea,
-} from '~/components/ui/text-field'
-import { MagicMoveElement } from 'shiki-magic-move/types'
+  Dialog,
```

Comment:
Maybe disable import sorting for now because this makes the diff harder to parse.